src/decorators/gb.ts

```diff
@@ -81,9 +83,19 @@ const decorator: IDecorator = {
       areaCodeLength = 5;
     }
 
+    let isExtension = false;
+
     for (let i = 0; i < phoneNumber.length; ++i) {
       const digit = phoneNumber[i];
-      decoratedNumber.parts.push(numberPart(digit));
+
+      if (digit === "x") {
+        decoratedNumber.parts.push(extensionDecorativePart(" ext. "));

```

Comment:
Part of this issue with the extensions stuff is that there isn't a well defined convention for how to display it. `ext.` seems to be used be some people, some places, but it only works in English.

German numbers do have a well defined format for displaying numbers.

I think the most sensible option is passing in the delimiter when you instantiate the library.

Reply:
My comment above is also in line with what Chris says here. I think the delimiter and replacement should be provided on instantiation

Reply:
Ok, I think that makes sense as well.